client/payment_test.go

```diff
@@ -98,3 +104,54 @@ func TestHappyAliceBob(t *testing.T) {
 
 	log.Info("Happy test done")
 }
+
```

Comment:
The comments in this file are missing a period at the end of the sentence.

Reply:
The code was just moved. The intention is not to cleanup existing code here. Just to satisfy the linter with regards to function length.